data/json/npcs/factions.json

```diff
@@ -24,6 +24,31 @@
     "mon_faction": "player",
     "description": "The survivors who have entrusted you with their well-being.  If morale drops, poor performance and mutiny may become issues."
   },
+  {
+    "type": "faction",
+    "id": "amf",
+    "name": "AMF",
+    "likes_u": -60,
+    "respects_u": -60,
+    "known_by_u": false,
+    "size": 1,
+    "power": 100,
+    "food_supply": 0,
+    "wealth": 0,
+    "relations": {
+      "your_followers": { "kill on sight": true, "knows your voice": true },
+      "amf": {
+        "kill on sight": false,
+        "watch your back": true,
+        "share my stuff": true,
+        "guard your stuff": true,
+        "lets you in": true,
+        "defends your space": true,
+        "knows your voice": true
+      }
+    },
+    "description": "The survivors who formerly entrusted you with their well-being, but have now mutinied because of your abuse and incompetenece.  They will kill you if they can."
```

Comment:
```suggestion
    "description": "The survivors who formerly entrusted you with their well-being, but have now mutinied because of your abuse and incompetence.  They will kill you if they can."
```